test/controllers/PrepareUploadControllerSpec.scala

```diff
@@ -24,39 +24,54 @@ class PrepareUploadControllerSpec extends UnitSpec with Matchers {
 
       "build and return upload URL if valid request" in {
 
+        //given
+
         val request: FakeRequest[JsValue] = FakeRequest().withHeaders("Content-type" -> "application/json")
```

Comment:
I think content-type would be inferred from `withBody(Json.obj`  but doesn't hurt